src/components/Carousel/Carousel.js

```diff
@@ -109,7 +98,7 @@ class Carousel extends Component {
 	};
 
 	render() {
-		const { images, thumbnails } = this.props;
+		const { images, thumbnails, action } = this.props;
```

Comment:
I think we're not using props 'action' here. We can remove this one.

Reply:
this has been address with latest commit